src/index.html

```diff
@@ -7,8 +7,87 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="use-credentials">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <div class="class"> -->
+    <header class="header">
+      <img
+          class="logo"
+          src="./images/logo.png" alt="">
+      <nav class="navBig">
+        <ul class="list">
+          <li class="is-active">
+            <a href="">apple</a>
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">apple</a>
```
не залишай поле href пустим.
Використовуй таку структуру.
Клас is-active має бути на посиланні, а не на `li`
стилізувати треба саме посилання